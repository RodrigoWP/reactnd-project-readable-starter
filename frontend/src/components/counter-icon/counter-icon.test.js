import React from 'react'
import { shallow } from 'enzyme'
import { CounterIcon } from './index'

it('Should render correctly with out props', () => {
  const component = <CounterIcon />
  expect(shallow(component)).toMatchSnapshot()
})

it('Should render correctly with count', () => {
  const component = <CounterIcon count={1} />
  expect(shallow(component)).toMatchSnapshot()
})
