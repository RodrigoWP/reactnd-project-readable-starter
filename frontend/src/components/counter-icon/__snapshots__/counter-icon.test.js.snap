// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly with count 1`] = `
<div>
  <label>
    1
  </label>
  <img />
</div>
`;

exports[`Should render correctly with out props 1`] = `""`;
